server/src/main/java/io/druid/server/initialization/ServerConfig.java

```diff
@@ -68,6 +70,15 @@
   @NotNull
   private Period unannouncePropogationDelay = Period.ZERO;
 
+  @JsonProperty
+  @Min(0)
+  private int inflateBufferSize = 4096;
```

Comment:
Can you make 4096 a static constant in ServerConfig, maybe DEFAULT_GZIP_INFLATE_BUFFER_SIZE

Reply:
+1 constant added